10up/gutenbridge
# Adjust enable_block_editor to prevent removal of block editor support

### Description of the Change
Instead of disabling the block editor if a post type doesn't support the gutenbridge, it will respect its current settings. This fixes an issue with this plugin removing Gutenberg support from post types with the block editor enabled.

Issue: https://github.com/10up/convert-to-blocks/issues/135

### How to test the Change

1. Create a WordPress website and install this plugin.
2. Disable support for the "post" post type. 

The block editor should still be visible when editing a post.

3. Enable the "Classic Editor" plugin

The classic editor should be visible with the block editor removed.

4. Enable plugin support for the "post" post type

The block editor should be visible when editing a post.

### Changelog Entry
> Changed - Adjust enable_block_editor method only to alter posts that support the gutenbridge.

### Credits
Not sure but on the issue @jeffpaul  and @dsawardekar confirmed this was an issue and agreed with the initial approach I had so thank you!

### Checklist:
